src/pages/common/Company.js

```diff
@@ -0,0 +1,124 @@
+import React, { Component } from 'react';
+import CompanyItem from '../../components/common/CompanyItem';
+import callApi from '../../utils/apiCaller';
+import loading_gif from './../../image/loader.gif';
+
+class Company extends Component{
+    constructor(props) {
+        super(props);
+        this.state={
+            "name": "",
+            "list_employer": [],
+            "count_page": 0,
+            "page": 1,
+            "loading": false
+        };
+    }
+    searchCompany = (e) => {
+        e.preventDefault();
+        this.setCountPage();
+        this.setPage(1);
+    }
+    setPage = (page) => {
+        this.setState({"page": page});
+        this.setState({"list_employer": []});
+        this.setListCompany(page);
+    }
+    setListCompany = (page) => {
+        this.setState({loading: true});
+        callApi("employer?name=" + this.state.name + "&page=" + (page - 1), 'GET').then(rs => {
```

Comment:
define a format string

Reply:
In Reactjs, format string need to import library. I think it's not effective in this case.